runtime/doc/intro.txt

```diff
@@ -71,12 +71,12 @@ If Nvim crashes, try to get a backtrace.  See |debug.txt|.
 Sponsor Vim/Nvim development				*sponsor* *register*
 
 Fixing bugs and adding new features takes a lot of time and effort.  To show
-your appreciation for the work and motivate Bram and others to continue
+your appreciation for the work and motivate Vim developers to continue
```

Comment:
That's the only thing I'm not 100% sure about; I'd prefer not to write anything on somebody else's behalf -- certainly not the Vim developers, who haven't made any official statements yet. Maybe just combine these two paragraphs and end this sentence after "show your appreciation for the work of Bram"?

(Or just not touch this section at all and wait what Vim will write here, once they have sorted everything out. There's no reason to rush this.)

Reply:
I went back and forth on changes to this section. I'll leave it untouched for now, and the new section can be the extent of the changes

Reply:
Removed